script.module.simplecache/README.md

```diff
@@ -0,0 +1,85 @@
+# script.module.simplecache
+A simple object cache for Kodi addons
+
+
+## Usage
+
+You can use this python library as module within your own Kodi scripts/addons.
+Just make sure to import it within your addon.xml:
+
+```xml
+<requires>
+    <import addon="script.module.simplecache" version="1.0.0" />
+</requires>
+```
+
+Now, to use it in your Kodi addon/script, make sure to import it and you can access it's methods.
+
+```
+import simplecache
+
+#get data from cache
+mycache = simplecache.get("MyChunkOfData")
```

Comment:
The internals are all prefixed/unique but good idea indeed to have a prefixed example. I'll change it right away.